src/providers/k8s/dynResourceDefinition.ts

```diff
@@ -106,7 +106,15 @@ function make_transfer_containter(): any {
     command: [
       "ash",
       "-c",
-      `until [ -f ${FINISH_MAGIC_FILE} ]; do echo waiting for tar to finish; sleep 1; done; echo copy files has finished`,
+      [ "wget https://github.com/moparisthebest/static-curl/releases/download/v7.83.1/curl-amd64 -O /cfg/curl",
```

Comment:
Why do you bother downloading `curl` when you have `wget` already ?

Reply:
Hi @chevdor, this is an `init` container since the polkadot image don't have `wget` or `curl` and the image run as non-root user I make `curl` available through `init containers`.